### PR TITLE
add a how-to explainer for the redirects page

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/5747 by adding a text explanation to the redirects page, above the redirects themselves.:

image

Live URL: https://foundation-s-redirect-e-memska.herokuapp.com/cms/redirects/

### CODE REVIEW

File: network-api/networkapi/wagtailcustomization/templates/wagtailredirects/index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <p>
+                Wagtail Redirects are rules for catching 404 errors, also known as "page not found" errors, and instead directing
+                users to another page, or external website. Redirects in the Mozilla Foundation CMS are locale-agnostic, meaning
+                that a redirect for any non-existent url <code>/some-page</code> will trigger regardless of whether it was requested
+                with a locale prefix. A rule for <code>/some-page</code> will automatically kick in for <code>/en/some-page</code>,
+                <code>/fr/some-page</code> etc. Also, note that redirects only work for pages that are <strong>not live</strong>.
+                You cannot create a redirect for a published page: you will have to unpublish that page before a redirect will work.
+            </p>
+            <hr/>
+        </div>
```
Comment (Pomax) [inline network-api/networkapi/wagtailcustomization/templates/wagtailredirects/index.html:72]: This is the only new code, the rest is a copy of https://github.com/wagtail/wagtail/blob/cf38b2457dba521d029d3a6e5024a7d0c3f5faf6/wagtail/contrib/redirects/templates/wagtailredirects/index.html

Comment (Pomax) [inline network-api/networkapi/wagtailcustomization/templates/wagtailredirects/index.html:72]: This is the only new code, the rest is a copy of https://github.com/wagtail/wagtail/blob/cf38b2457dba521d029d3a6e5024a7d0c3f5faf6/wagtail/contrib/redirects/templates/wagtailredirects/index.html

File: network-api/networkapi/wagtailcustomization/templates/wagtailredirects/index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    </div>
+                </div>
+            </div>
+        </header>
+    {% else %}
+        {% include "wagtailadmin/shared/header.html" with title=redirects_str icon="redirect" search_url="wagtailredirects:index" %}
+    {% endif %}
+
+    <div class="nice-padding">
+        <div id="redirects-explanation" class="">
```
Comment (Pomax) [inline network-api/networkapi/wagtailcustomization/templates/wagtailredirects/index.html:59]:
```suggestion
        <div id="redirects-explanation">
```

**Review by mmmavis — APPROVED** [review]
Lovely!

**Review by taisdesouzalessa — APPROVED** [review]
Copy looks great to me: clear and highlighting the most important points. Thanks for working on that, Pomax!

**Pull request merged**